src/Symfony/Bridge/Twig/Tests/Command/LintCommandTest.php

```diff
@@ -107,6 +108,53 @@ public function testLintDefaultPaths()
         self::assertStringContainsString('OK in', trim($tester->getDisplay()));
     }
 
+    public function testLintIncorrectFileWithGithubFormat()
+    {
+        if (!class_exists(GithubActionReporter::class)) {
+            $this->expectException(\InvalidArgumentException::class);
+            $this->expectExceptionMessage('The "github" format is only available since "symfony/console" >= 5.3');
+        }
+
+        $filename = $this->createFile('{{ foo');
+        $tester = $this->createCommandTester();
+
+        $tester->execute(['filename' => [$filename], '--format' => 'github'], ['decorated' => false]);
+
+        if (!class_exists(GithubActionReporter::class)) {
+            return;
+        }
+
+        self::assertEquals(1, $tester->getStatusCode(), 'Returns 1 in case of error');
+        self::assertStringMatchesFormat('%A::error file=%s, line=1, col=0::Unexpected token "end of template" ("end of print statement" expected).%A', trim($tester->getDisplay()));
+    }
+
+    public function testLintAutodetectsGithubActionEnvironment()
+    {
+        if (!class_exists(GithubActionReporter::class)) {
+            $this->markTestSkipped('The "github" format is only available since "symfony/console" >= 5.3.');
+        }
+
+        $prev = getenv('GITHUB_ACTIONS');
+        putenv('GITHUB_ACTIONS');
+
+        try {
+            putenv('GITHUB_ACTIONS=1');
+
+            $filename = $this->createFile('{{ foo');
+            $tester = $this->createCommandTester();
+
+            $tester->execute(['filename' => [$filename]], ['decorated' => false]);
+
+            if (!class_exists(GithubActionReporter::class)) {
```

Comment:
this is a dead condition. The class with always exist here as the test is skipped otherwise.